pkgs/development/tools/protoc-gen-grpc-java/Makefile

```diff
@@ -0,0 +1,28 @@
+CXX ?= g++
```

Comment:
Where did you get that from?

Reply:
I copied the makefile in `protoc-gen-grpc-web` and adapted it slightly. :slightly_smiling_face:

Reply:
I think we should at least remove the absolute paths which obviously can't wokr.

Reply:
I've removed the absolute paths, and included a comment that references the source.